### ISSUE TITLE
Opinions needed for Ghostery Dawn

### ISSUE THREAD

**Issue by chrmod** [yokoffing/Better-Fox#4]
Hi, my name is Krzysztof and I'm developing Ghostery Dawn - Privacy focused Firefox fork coming from Ghostery.

Our community member @PoorPocketsMcNewHold brought your project to our attention https://github.com/ghostery/user-agent-desktop/issues/458. 
It's great to see such nicely put set of Firefox configs in one place. We will gladly copy those into Dawn project. 

It would be even better if we can get your opinion on the configs we've put together already https://github.com/ghostery/user-agent-desktop/blob/a17fe90d769324ce4e42712168d7da2084dac909/brands/ghostery/branding/pref/firefox-branding.js and what we could improve. Would you like to collaborate with us on creating those configs?

**Comment by chrmod** [comment]
I can imaging `Better-Fox` to have a section Ghostery Dawn `firefox-branding.js` file or even its own dedicated file. Ghostery could feature `Better-Fox` in Readme and release blog posts so give this project more visibility.

**Comment by yokoffing** [comment]
Hi Krzysztof! Thank you for reaching out and for your kind words.

I joined the beta testing for the Ghostery [desktop browser](https://www.ghostery.com/products/ghostery-browser/) a few weeks back. Is this the same thing as Ghostery Dawn? I haven't opened that browser since the first week of the launch of beta testing. I was underwhelmed with the lack of `about:config` changes out of the box, but I acknowledge it was only the first week of beta testing. It looks like you all have made progress since the initial launch!

To comment specifically on the configs you [referenced](https://github.com/ghostery/user-agent-desktop/blob/a17fe90d769324ce4e42712168d7da2084dac909/brands/ghostery/branding/pref/firefox-branding.js): From a cursory reading, nothing you all have introduced should cause breakage. It only seems like there needs to be some categorization of the preferences so you can organize them appropriately. Then move forward with adding more preferences. There's so many preferences with Firefox — they become unwieldy over time as you add more and more.

I would be more than happy to collaborate with you all on those configs as I have time. This is a side project ("passion project") for me, so my job and education would come first, of course. Just let me know what we need to do to more forward.

`Better-Fox` having a section/file in Ghostery Dawn would amazing. This alone would be more recognition than I thought this project would ever get — much less a dedicated spot in the `Readme` or blog posts. It would be spectacular. And who knows? Maybe I can join a [podcast](https://open.spotify.com/show/1kMddrJxxueyEwsoQWMW3c?si=GHFwawkfQ4WnrOppDtwoww) episode, too. (Just kidding 🙄😂)

P.S. Last year I built a page dedicated for Ghostery so users could have an understanding of how the extension works. I created it based on my exchanges with Remi ([@remusao](https://github.com/remusao)) and on his comments on Reddit regarding Ghostery's functionality; I also added a few other primary sources directly from Ghostery and CLIQZ. You can find that [here](https://github.com/yokoffing/Better-Fox/wiki/Ghostery).

**Comment by chrmod** [comment]
@yokoffing I'm super happy to hear that :-) Will prepare the integration in upcoming days and let you know on how we can move forward.

**Comment by yokoffing** [comment]
That sounds great!  I've created a draft of the prefs to import into Ghostery Dawn to get us started. I will tweak them a little more today.

We would not use the [BetterFox user.js](https://github.com/yokoffing/Better-Fox/blob/master/user.js) as-it-is since Ghostery Dawn disables a lot of Firefox services (like Enhanced Tracking Protection and Lockwise). What's more, the user.js itself disables some standard functionalities of the browser (like built-in password manager) that we _want_ to keep for Ghostery Dawn widespread deployment. I've already made adjustments for considerations such as those.

If we need to video conference to get everything set up initially and to discuss why I chose some prefs over others, or any other needs for clarification, I am open to that as well.
